app/e2e/common/global/test-setup.ts

```diff
@@ -0,0 +1,24 @@
+import { chromium } from '@playwright/test';
+import { PlaywrightLandingPage } from '@local/common/pages/playwright-landing-page';
+
+async function globalSetup() {
+    // Setup Context
+    const browser = await chromium.launch();
```

Comment:
Yes because in this case it's not a test, it's simply part of the setup. If you launched multiple browsers here, it would fail on all but one since only one could create the account and the rest would all get errors since the email would be the same.

Reply:
I see, thanks for the explanation.

I feel like maybe we should be able to test multiple browsers at once -- maybe if data belonged to each individual test, but maybe that would lead to the same problem. Outside the scope of this PR though.